torchok/constructor/config_structure.py

```diff
@@ -134,17 +134,12 @@ class TaskParams:
 # Trainer parameters
 @dataclass
 class TrainerParams:
-    enable_checkpointing: bool = True
-    default_root_dir: Optional[str] = None
-    gradient_clip_val: Optional[float] = None
-    gradient_clip_algorithm: Optional[str] = None
+    accelerator: str = "auto"
+    strategy: str = "auto"
+    devices: str = "auto"
```

Comment:
Shouldn't be the example configs updated after all these changes?

Reply:
No, they are compatible